SJSU-1035_3, History Page

SandroSal opened the pull request:
new styling for History page.

what I changed:
- cancellationPrompt.js : just modified the dates, since I changed the format they are passed
- HistoryExpansionTable.js : broke this part out of the main component
- HistoryOverview.js : added rewards points overview, most other changes is just shifting code around
- history.css : added min-height so it won't display the footer in the middle of the page
- profileReducer: took out the dummy values

DaleCS commented on front_end/src/components/history/history.css:
```diff
@@ -235,3 +253,12 @@
+  font-weight: 600;
+  margin-bottom: 0px;
+  margin-right: 25px;
+}
```
Couldn't highlight it but there is a class named .HistoryPageDates and another called .HistoryContainerExpand both aren't being used.

DaleCS commented on front_end/src/components/history/HistoryOverview.js:
```diff
@@ -1,26 +1,26 @@
+// - IF CHANGE, SHOW OLD/NEW DATES
+
 import React, { Component } from "react";
 import { withStyles, withTheme, withWidth } from "@material-ui/core/styles";
```
withTheme and withWidth were imported but never used.

SandroSal replied:
removed

SandroSal commented:
- added code so the panel won't expand on change-button-click
- buttons are part of other stories. They will be designed by Yu and Minh respectively.
- removed the unused imports
